cheroot/server.py

```diff
@@ -1146,8 +1196,8 @@ def send_headers(self):
                 pass
             else:
                 needs_chunked = (
```

Comment:
You don't need to handle this yourself anymore – if you don't have a `Content-Length`, then h11 will automatically choose between HTTP/1.1-style chunked encoding and HTTP/1.0-style connection-termination for you based on the capabilities of the peer, and add/remove `Transfer-Encoding: chunked` and `Connection: close` headers as appropriate.